ruby.html.markdown

```diff
@@ -206,6 +207,8 @@ new_hash.has_value?(3) #=> true
 # They share a lot of useful methods such as each, map, count, and more
 
 # Control structures
+# Ruby has the special 'unless' statement, which is syntactic sugar.
+# Unless (some condition) = If !(some condition).
```

Comment:
I would rather put this into words, rather than with `=` signs:

```ruby
# Equivalent statements:
#  unless some_condition
#  if !some_condition
```